src/test/scala/cqrs/domain/orders/DomainTest.scala

```diff
@@ -2,6 +2,8 @@ package cqrs
 package domain.orders
 
 import java.util.UUID
+import cqrs.domain.orders.OrderCommandHandler.UnknownOrderException
```

Comment:
this import needs to be on line 18 and you can change it to:
`import OrderCommandHandler.UnknownOrderException`